### PR TITLE
Add swedish language

### PR DESCRIPTION
**Pull request by Sjoo90**

### CODE REVIEW

File: src/locales/sv_SE.json [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                "show_window": "Visa fönster",
+                "toggle_minimize": "Växla minimerat till aktivitetsfält",
+                "start_minimize": "Starta minimerat till aktivitetsfält",
+                "close": "Stäng till aktivitetsfält"
+            },
+            "escape": {
+                "title": "Escape beteende",
+                "hide": "Göm fönster",
+                "clear": "Rensa fält",
+                "default": "default when tray is not showing"
```
Comment (averissimo) [inline src/locales/sv_SE.json]: missing translation

**Comment by Sjoo90** [comment]
Now added missing translation.

**Comment by averissimo** [comment]
very nice work!! I always enjoy looking at a language that I don't understand :)

**Comment by averissimo** [comment]
pre-release should be updated in 10-20 minutes with your work

**Comment by averissimo** [comment]
arghhh, I forgot one thing, the language name should be in swedish instead of english, to be in line with other translations

Can you confirm that is 'Svenska'?

**Comment by Sjoo90** [comment]
Yes that's correct

**Comment by averissimo** [comment]
cool, i've pushed it directly and new binaries have been built! :)

it is already available in pre-release v1.4.4

**Pull request merged**